index.js

```diff
@@ -104,6 +104,7 @@
 
   //  identity :: a -> a
   function identity(x) { return x; }
+  Object.freeze(identity);
```

Comment:
I concede. If a lib/user wants to add properties to `identity` they're doing it wrong.
> Would this even cause problems?

Probably not. I'm just being overly cautious.